### PR TITLE
[processing] Project cartesian algorithm Fix #2370

### LINKED ISSUE #2370: [FEATURE][processing] Add 'Project points (cartesian)' algorithm
Original commit: https://github.com/qgis/QGIS/commit/1bada065ed7c40958c443af79918434d75b28e04 by nyalldawson

Projects points from an input point layer by a specified distance
and bearing (azimuth). Supports dynamic parameters for the distance
and bearing so that they can use field values or expressions.

### PR DESCRIPTION
**Pull request by ghtmtt**
Fix #2370

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 1]
```diff
@@ -1467,6 +1467,35 @@ Output
...
+a new point layer with the projected points.
+
+Parameters
+..........
+
+``Input layer`` [vector: point]
+  Point vector layer to project
+
+``Bearing (degrees from North)`` [number]
+  Clockwise degrees starting from North
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: `Clockwise angle starting from North, in degree (`°`) unit`?
I don't know if angle is a better synonym for that but...

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 2]
```diff
@@ -1467,6 +1467,35 @@ Output
...
+
+``Distance`` [number]
+  Distance in layer units
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: `Distance to offset geometries, in layer units`?

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 3]
```diff
@@ -1467,6 +1467,35 @@ Output
...
+
+Output
+......
+
+``Projected`` [vector: point]
+  Projected layer at given degrees and distance
+
+
+
+The distance is specified in layer units, and the bearing in degrees clockwise from North.
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: I'd remove this line as the layer units is already in distance parameter description and add the `°` units information to bearing parameter (as proposed above) AND/OR move it in the alg description itself.

Comment (ghtmtt) [reply source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: oh sorry for that.. an orphan line ;)

**Pull request merged**